src/runtime/components/navigation/CommandPalette.vue

```diff
@@ -315,10 +313,9 @@ export default defineComponent({
     // Methods
 
     function activateFirstOption () {
-      // hack combobox by using keyboard event
-      // https://github.com/tailwindlabs/headlessui/blob/6fa6074cd5d3a96f78a2d965392aa44101f5eede/packages/%40headlessui-vue/src/components/combobox/combobox.ts#L769
       setTimeout(() => {
-        comboboxInput.value?.$el.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowDown' }))
+        // https://github.com/tailwindlabs/headlessui/blob/6fa6074cd5d3a96f78a2d965392aa44101f5eede/packages/%40headlessui-vue/src/components/combobox/combobox.ts#L804
+        comboboxInput.value?.$el.dispatchEvent(new KeyboardEvent('keydown', { key: 'PageUp' }))
```

Comment:
Will `PageUp` work the same as `ArrowDown`?

Reply:
I think we need to keep `ArrowDown` with `props.autoselect` and use `PageUp` when query changes.

Reply:
`PageUp` will always activate the first item, while `ArrowDown` will activate the next item. However, if the first item is already activated, `ArrowDown` will activate the second one

Reply:
Yes I think we can keep ArrowDown with `props.autoselect`